eng/pipelines/corefx-base.yml

```diff
@@ -55,6 +59,9 @@ jobs:
     - template: ../common/templates/job/job.yml
       parameters:
         variables:
+        - name: _dotnetFeedUrl
+          value: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
```

Comment:
Sorry for going back and forth on this, for variable names, we use the following syntax:

- variableName: value -- `- _dotnetFeedUrl: https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json`

Reply:
No need, to do an official build for this, if the PR builds compile and run correctly, then the syntax is correct.

Reply:
Also, since this is just used in official builds you could move it down to the if statement where you declare 
`-group: DotNet-Blob-Feed` 

https://github.com/dotnet/corefx/pull/35473/files#diff-4aae51df678fdaa7a5e0769b0593310aR90

Reply:
FYI, publish.yml uses the value/name syntax for all of it's variables.  It is where I copied _dotnetFeedUrl in the first place.

Reply:
publish.yml uses that syntax because it doesn't use the Job.yml template from arcade, so the cool `-key: value` syntax is not available for that yml file 😄